<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>–ñ—É—Ä–Ω–∞–ª –∏–∑–º–µ–Ω–µ–Ω–∏—è –≤–µ—Å–∞</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>–î–∞—Ç–∞</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>–í–µ—Å</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLineEdit" name="dateEdit">
        <property name="placeholderText">
         <string>–í—ã–±–µ—Ä–∏—Ç–µ –¥–∞—Ç—É</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="calendarButton">
        <property name="text">
         <string>üìÖ</string>
        </property>
        <property name="toolTip">
         <string>–í—ã–±—Ä–∞—Ç—å –¥–∞—Ç—É –∏–∑ –∫–∞–ª–µ–Ω–¥–∞—Ä—è</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="weightEdit">
        <property name="placeholderText">
         <string>–í–≤–µ–¥–∏—Ç–µ –≤–µ—Å (–∫–≥)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>–î–æ–±–∞–≤–∏—Ç—å</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>–°–æ—Ö—Ä–∞–Ω–∏—Ç—å</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>–ó–∞–≥—Ä—É–∑–∏—Ç—å</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>–£–¥–∞–ª–∏—Ç—å –≤—ã–±—Ä–∞–Ω–Ω—É—é –∑–∞–ø–∏—Å—å</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
